<?xml version="1.0"?>

<launch>
	<arg name="port_name" default="/dev/ttyCH343USB0" />
	<arg name="baud_rate" default="921600" />
	
    <node pkg="nlink_parser" type="linktrack" name="linktrack0" output="screen">
        <param name="port_name" value="$(arg port_name)" />
        <param name="baud_rate" value="$(arg baud_rate)" />
    </node>
    
</launch>
